person.rb

```diff
@@ -1,19 +1,16 @@
-class Nameable
-  def correct_name
-    raise NotImplementedError, 'You must implement the correct_name method.'
-  end
-end
+require_relative 'nameable'
+require_relative 'rental'
 
-class Person < Nameable
+class Person

```

Comment:
- [ ] The Person class should have the method for association with the Book class.

  ```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ```

- [ ] **[OPTIONAL]** I think you should reinstate the inheritance from the Nameable class you had earlier.

  ```
  class Person < Nameable
  ```